Sources/Files.swift

```diff
@@ -685,6 +685,15 @@ public final class Folder: FileSystem.Item, FileSystemIterable {
     public func containsFile(named fileName: String) -> Bool {
         return (try? file(named: fileName)) != nil
     }
+
+    /**
+     *  Return whether this folder contains a given file
+     *
+     *  - parameter file: The file to check for
+     */
+    public func contains(_ file: File) -> Bool {
+        return files.contains(file)
```

Comment:
The problem with this approach is that it's `O(N)`, since it performs a linear search through the underlying `FileSystemSequence`. Instead, I think it's better to do what `containsFile(named:)` does and attempt to construct a `File` instance and return `false` otherwise.

Reply:
Great point. I changed the implementation to defer to `containsFile(named:)`.